rasa/shared/core/training_data/visualization.py

```diff
@@ -42,6 +42,8 @@
 
 VISUALIZATION_TEMPLATE_PATH = "/visualization.html"
 
+logger = logging.getLogger(__name__)
```

Comment:
I don't see the `logger` being used 🤔 can you remove it? (or add some debug, info etc. statements)